ncm-postfix/src/main/pan/components/postfix/schema.pan

```diff
@@ -507,6 +507,7 @@ type postfix_main = {
     "smtpd_recipient_restrictions" ? string[]
     "smtpd_reject_udicted_recipient" ? boolean
     "smtpd_reject_udicted_sender" ? boolean
+    "smtpd_relay_restrictions" ? string[]
```

Comment:
Is this something that can be validated, or is it too complex?

Reply:
I'm not 100% sure what values are all allowed here and we don't do any validation for any of the similar options (like `smtpd_recipient_restrictions`).